crates/nostr/src/event/builder.rs

```diff
@@ -1679,6 +1680,21 @@ impl EventBuilder {
         ])
     }
 
+    /// User Statuses
+    ///
+    /// <https://github.com/nostr-protocol/nips/blob/master/38.md>
+    #[inline]
+    pub fn live_statuses<S, I>(status: LiveStatus, content: S, extra_tags: I) -> Self
```

Comment:
I would rename it to `live_status`, what do you think?
And since any extra tag can be added with `EventBuilder::tags` I think is better to remove the `extra_tags` arg.